tests/parsers/appbio_quantstudio/testdata/appbio_quantstudio_example05.json

```diff
@@ -540,7 +540,7 @@
                                                 "value": 0.06,
                                                 "unit": "(unitless)"
                                             },
-                                            "automatic cycle threshold enabled setting": true,
+                                            "automatic cycle threshold enabled setting": false,
```

Comment:
NOTE to reviewer: these changes are due to a bug in the existing version, fixed here.